leakcanary-analyzer/src/test/java/com/squareup/leakcanary/HeapAnalyzerTest.java

```diff
@@ -0,0 +1,56 @@
+package com.squareup.leakcanary;
+
+import com.squareup.haha.perflib.RootObj;
+import com.squareup.haha.perflib.Snapshot;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import static com.squareup.haha.perflib.RootType.SYSTEM_CLASS;
+import static java.util.Arrays.asList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class HeapAnalyzerTest {
+  public static final List<RootObj> DUP_ROOTS =
+          asList(new RootObj(SYSTEM_CLASS, 6),
+                  new RootObj(SYSTEM_CLASS, 5),
+                  new RootObj(SYSTEM_CLASS, 3),
+                  new RootObj(SYSTEM_CLASS, 5),
```

Comment:
Should add an entry with same id but different type, and have a test that ensures it stays here.